crypto/o_time.c

```diff
@@ -47,6 +47,35 @@ typedef struct _ile3 {          /* Copied from ILEDEF.H for Alpha   */
 # endif   /* alpha || ia64    */
 #endif    /* OPENSSL_SYS_VMS  */
 
+time_t OPENSSL_timegm(struct tm *tm)
+{
+    if (1) {
+#if defined(__USE_MISC)
+        return timegm(tm);
+    } else {
+#endif
+        /* Figures it out without mucking the environment */
+        time_t currtime, utc, local;
+        struct tm *timeinfo;
+        int offset;
+
+        time(&currtime);
```

Comment:
Thinking about it more, I agree. If I don't use the current time, I would use something a bit more positive than 1970-01-01 to avoid negative dates.